`print ("got color map\n");` removed from `editor/pnmquantall` so remove from test
Do this from the patch instead of ebuild because if upstream also fixes it the patch will fail, sed would not

--- a/test/pnmquantall.ok
+++ b/test/pnmquantall.ok
@@ -1,4 +1,3 @@
-got color map
 2373957371 33838
 3892560659 33838
 1383839923 33838
